SPECS-EXTENDED/libosinfo/libosinfo.spec

```
@@ -0,0 +1,323 @@
+Summary:        A library for managing OS information for virtualization
+Name:           libosinfo
+Version:        1.10.0
+Release:        2%{?dist}
+License:        LGPLv2+
```

Mind changing this to an [SPDX expression](https://spdx.org/licenses/)? We're slowly moving away from Fedora short identifiers.

For future reference, [this browser plugin](https://github.com/spdx/spdx-license-diff) is pretty useful for figuring out the identifier for ambiguous licenses (especially the various BSD/MIT variants).

```suggestion
License:        LGPL-2.1-or-later
```